blobfuse/src/Permissions.cpp

```diff
@@ -36,8 +36,7 @@ mode_t aclToMode(access_control acl)
     }
     else if(permissions.size() != blobfuse_constants::acl_size)
     {
-        syslog(LOG_ERR, "Failure: Unexpected amount of permissions from service : %s", permissions.c_str());
-        return mode;
+        syslog(LOG_DEBUG, "Unexpected amount of permissions from service : %s", permissions.c_str());
```

Comment:
Check if it is sticky. Ignore if it is sticky. Issue LOG_WARNING otherwise.

Reply:
From blobfuse we can not set the sticky bit or anything other then 'rwx' triplet so we are not sure for what reason this extra permissions are coming or how the user was able to set them. Anyway while setting or reading the permissions blobfuse only parses 'rwx' triplet (first 9 bytes) so even if these extra bits come, we do not handle those. Have put a debug log just to get info on what these extra permissions are, just in case we face any issue with parsing or setting.